app/services/rooms_config_getter.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class RoomsConfigGetter
+  def initialize(setting_name:, provider:)
+    @setting_name = setting_name
+    @provider = provider
+  end
+
+  def call
+    config = MeetingOption.joins(:rooms_configurations)
+                          .select(:value)
+                          .where(rooms_configurations: { provider: @provider })
+                          .find_by(name: @setting_name)
+
+    config&.value
```

Comment:
I'm not sure having a service here makes sense. I think this should be an action in the model instead

Reply:
A method on the model?

Reply:
Yep